meta-xt-domx/conf/layer.conf

```diff
@@ -9,5 +9,5 @@ BBFILE_COLLECTIONS += "xt-domx"
 BBFILE_PATTERN_xt-domx := "^${LAYERDIR}/"
 BBFILE_PRIORITY_xt-domx = "6"
 
-LAYERSERIES_COMPAT_xt-domx = "dunfell"
+LAYERSERIES_COMPAT_xt-domx = "dunfell hardknott"
```

Comment:
In one of the following patches you drop support for Weston 8.
Will this layer be compatible with `dunfell` after that?

Reply:
That's a good point. I you're right. Unfortunately, because of weston, I can't leave dunfel compatibility.